tools/cli/src/helpers/errorBehavior.ts

```diff
@@ -0,0 +1,50 @@
+import { Err, Ok, Result, ResultErr } from '@backtrace-labs/sourcemap-tools';
+
+export const ErrorBehaviors = {
+    exit: 'exit',
+    error: 'error',
+    warn: 'warn',
+    info: 'info',
+    debug: 'debug',
+    trace: 'trace',
+    skip: 'skip',
+} as const;
+
+export type ErrorBehavior = keyof typeof ErrorBehaviors;
+
+export function GetErrorBehavior(type: string): Result<ErrorBehavior, string> {
+    const valid = Object.keys(ErrorBehaviors);
+    if (valid.includes(type)) {
+        return Ok(type as ErrorBehavior);
+    }
+
+    return Err(`invalid error behavior "${type}", expected one of: ${valid.join(', ')}`);
+}
+
+export interface FailedElement<E> {
```

Comment:
isn't FailedElement too generic? Wouldn't it be better if we rename it to FailedErrorBehavior?

Reply:
Renamed to `BehaviorSkippedElement`.